src/plots/gl2d/convert.js

```diff
@@ -123,7 +123,9 @@ proto.merge = function(options) {
 
             this.tickEnable[i+j] = false;
             this.tickColor[i+j] = str2RGBArray((ax.tickfont || {}).color);
-            this.tickAngle[i+j] = (ax.tickangle === 'auto') ? 0 : -ax.tickangle;
+            this.tickAngle[i+j] = (ax.tickangle === 'auto') ?
+                0 :
+                Math.PI * -ax.tickangle / 180;
```

Comment:
If you're like me, I find this way more explicit than having some degrees-to-radians lib function.

But, I'd be ok with adding a lib function if preferable.